eg/temperature-mpu6050.js

```diff
@@ -1,17 +1,17 @@
-var five = require("../lib/johnny-five.js");
-var board = new five.Board();
+const {Board, Thermometer} = require("../lib/johnny-five.js");
+const board = new Board();
 
-board.on("ready", function() {
-  var thermometer = new five.Thermometer({
+board.on("ready", () => {
+  const thermometer = new Thermometer({
     controller: "MPU6050"
   });
 
-  thermometer.on("change", function() {
-    console.log("Thermometer");
-    console.log("  celsius      : ", this.celsius);
-    console.log("  fahrenheit   : ", this.fahrenheit);
-    console.log("  kelvin       : ", this.kelvin);
-    console.log("--------------------------------------");
+  thermometer.on("change", () => {
+    console.log(`Thermometer
+  celsius      : ${thermometer.celsius}
+  fahrenheit   : ${thermometer.fahrenheit}
+  kelvin       : ${thermometer.kelvin}
+--------------------------------------`);
```

Comment:
Next update should include using `common-tags` module so these can be: 

```js
  thermometer.on("change", () => {
    console.log(
      st.stripIndent`Thermometer
        celsius      : ${thermometer.celsius}
        fahrenheit   : ${thermometer.fahrenheit}
        kelvin       : ${thermometer.kelvin}
        --------------------------------------`
    );
  });

```
for its `ct.stripIndent&#96;...&#96;` template tag

Reply:
But now we have some that were left using `console.log(multi, arg, style)` and some with template strings :|

Reply:
Oh, I thought I got the multi arg ones. I'll look again and I could go ahead and add common-tags to this PR.